packages/dx-react-scheduler/src/plugins/appointments.jsx

```diff
@@ -26,21 +28,32 @@ export class Appointments extends React.PureComponent {
         <Template
           name="appointment"
         >
+          {params => (
```

Comment:
This function is recreated on each `render` though does not use anything from closure - it can be declared once.

Reply:
We use `containerComponent` from `props` here

Reply:
We use `resizeComponent` from props here.

```js
const Templates = (params, Container) => (
  <Container style={params.style}>
    <TemplatePlaceholder name="appointmentTop" params={{ data: params.data, type: params.type, slice: params.leftSlice }} />
    <TemplatePlaceholder name="appointmentContent" params={params} />
    <TemplatePlaceholder name="appointmentBottom" params={{ data: params.data, type: params.type, slice: params.rightSlice }} />
  </Container>
);
```


And use Templates(params, Container) in the `render` method. Is the function above better than current way?

Reply:
My fault - I didn't notice `Container.